test/browser/components.js

```diff
@@ -151,12 +151,6 @@ describe('Components', () => {
 		expect(scratch.innerHTML).to.equal('');
 	});
 
-	// Test for #651
-	it('should set enumerable boolean attribute', () => {
```

Comment:
I just moved this into the render.js UT file since this test doesn't actually use an components